internal/docker/docker.go

```diff
@@ -262,6 +264,7 @@ func (d *Client) start(ctx context.Context, req *Request) (string, error) {
 			},
 			Mounts:       req.Mounts,
 			PortBindings: req.PortBindings,
+			AutoRemove:   req.AutoRemove,
```

Comment:
How does this interact with IMAGETEST_SKIP_TEARDOWN?

Reply:
This is just the equivalent of `docker run ... --rm`, so in the case where IMAGETEST_SKIP_TEARDOWN is specified it just means any associated volumes/container when its stopped

Reply:
Ah, makes sense.